test/uk/gov/hmrc/gform/sharedmodel/JsonParseTestFormat.scala

```diff
@@ -181,4 +181,52 @@ class JsonParseTestFormat extends Spec with TableDrivenPropertyChecks {
     }
   }
 
+  "A text area component with a valid no of rows" should "parse correctly" in {
+
+    val multilineCombinations = Table(
+      ("rows", "expected"),
+      (7, TextArea(ShortText.default, Value, rows = Some(7))),
+      (10, TextArea(ShortText.default, Value, rows = Some(10))),
+      (40, TextArea(ShortText.default, Value, rows = Some(40)))
+    )
+
+    forAll(multilineCombinations) { (rows, expected) ⇒
+      val jsResult =
+        implicitly[Reads[FormComponent]]
+          .reads(Json.parse(startOfJson + s""", "format" : "shortText", "multiline" : "yes", "rows" : $rows }"""))
+
+      jsResult shouldBe a[JsSuccess[_]]
+      jsResult.map(fv => fv.`type` shouldBe expected)
+
+    }
+  }
+
+  "A text area component with an invalid no of rows" should "default to text area" in {
+
+    for {
+      snippet <- List(""", "format": "shortText", "multiline" : "yes", "rows" : "INVALID NO OF ROWS"}""")
```

Comment:
Why is this a list?

Reply:
Btw,  arent we expected to get an error instead of silently defaulting to None?

Reply:
Removed List. There is no error in case of invalid value for rows same as **multiline**. It will default to None.